pkg/nsx/endpoint.go

```diff
@@ -174,19 +174,19 @@ func (ep *Endpoint) KeepAlive() {
 }
 
 func (ep *Endpoint) setup() {
-	log.V(1).Info("begin to setup endpoint")
+	log.V(2).Info("begin to setup endpoint")
 	err := ep.keepAlive()
 	if err != nil {
 		log.Error(err, "setup endpoint failed")
 	} else {
-		log.V(1).Info("setup endpoint successfully")
+		log.V(2).Info("setup endpoint successfully")
 	}
 }
 
 func (ep *Endpoint) setStatus(s EndpointStatus) {
 	ep.Lock()
 	if ep.status != s {
-		log.Info("endpoint status is changing", "endpoint", ep.Host(), "oldStatus", ep.status, "newStatus", s)
+		log.V(2).Info("endpoint status is changing", "endpoint", ep.Host(), "oldStatus", ep.status, "newStatus", s)
```

Comment:
endpoint status is important. it's worth to  watch if updated. it's better to set it to Info

Reply:
ok